### ISSUE TITLE
Adding Lektor-icons theme

### ISSUE THREAD

**Issue by ikus060** [lektor/lektor-themes#4]
https://github.com/spyder-ide/lektor-icon

Would be nice to include this one here

**Comment by CAM-Gerlach** [comment]
Yep, sorry for the delay; I meant to add it two years ago once we ported the Spyder website over to v1.0 (the first version really fit for public consumption), but that never ended up happening until right now, during which time a number of other sites have employed it, and development has continued to the point of releasing v2.0 (which is fully backward compatible, so we're just porting it directly to that). However, the good news is that the porting process should be _finally_ be complete in the next few days, after which I'll cut the official v2.0 release and add it here. Thanks!
